src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ onTabSelected, tabs, selectedTabId }) => {
+  const setTab = tabs.find(el => el.id === selectedTabId) || tabs[0];
```

Comment:
`setTab` sounds like it should be a function, but it's a variable. So rename it 
```suggestion
  const selectedTab = tabs.find(el => el.id === selectedTabId) || tabs[0];
```